files/etc/dnsmasq.conf

```diff
@@ -8,6 +8,10 @@ expand-hosts
 # Never send any AAAA responses
 filter-AAAA
 
+# Domain + local (will not submit .local.mesh upstream) 
+domain=local.mesh
```

Comment:
You dont need this. This is defined in the auto-generated dnsmasq and comes from values defined in the UCI state. It was recently removed because it was redundant.